gartan_fetch.py

```
@@ -279,28 +279,20 @@ def gartan_login_and_get_session():
 
-    try:
-        # Attempt login with retry limit
-        form, resp = _get_login_form(session)
-        post_url = _get_login_post_url(form)
-        payload = _build_login_payload(form, username, password)
-        headers = _get_login_headers()
-
-        _post_login(session, post_url, payload, headers)
-        _get_data_page(session, headers)
+    # Attempt login with retry limit
+    form, resp = _get_login_form(session)
```

Ruff (0.14.8)

Unpacked variable `resp` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)